### ISSUE TITLE
Error uploading updated version of Topography

### ISSUE THREAD

**Issue by sitangshugk95** [ContactEngineering/topobank#960]
I had initially uploaded a non-periodic version of the Contact Mechanics Challenge topography, and I have now added a periodic version and uploaded. However, I am getting an error when trying to publish it. 
![image](https://user-images.githubusercontent.com/108830413/215830639-df36ad23-623f-4925-911b-426b0aa6d72e.png)
Could someone help with what this error means and how to fix it?
Is it possible to add measurements after it has been published? Or should I just create a new surface for the periodic version?

**Comment by pastewka** [comment]
Thanks for reporting @sitangshugk95. We pushed an update today and must have broken this. I won't manage to fix this today but will check tomorrow. Will probably ask you to try publishing again.

**Comment by sitangshugk95** [comment]
Thanks @pastewka , no worries. Please let me know when you fix it, and I can try uploading it again.

**Comment by mcrot** [comment]
@sitangshugk95 , published surfaces cannot be altered. There should be still the "Work in progress" version of the
published surface which was the blueprint for the published one. You can still change this blueprint, add measurements and publish again. Then you'll have a second published dataset with an increased version number. The first published dataset will still be there, as well as the "work in progress" version of the surface.

**Comment by sitangshugk95** [comment]
@mcrot yes, I have added the new measurement and it does show WIP: 
![image](https://user-images.githubusercontent.com/108830413/216100101-1bae940f-2834-4eb2-b70d-c59fea2ae640.png)

I then click on "Publish" which takes me to the following page:
![image](https://user-images.githubusercontent.com/108830413/216100421-b52c536c-fc2f-428c-89ee-397c8a9e5c9d.png)

Then I get the error that I have posted about originally. 

Basically, I am not being able to publish my second dataset, and it just shows work in progress (instead of an increased version number). I might just choose to create a new copy instead.

**Comment by sitangshugk95** [comment]
I tried creating an entirely new twin and publishing it - the same error shows. Looks like the issue is somewhere else (not related to updating the blueprint/WIP/version of twin)

**Comment by pastewka** [comment]
This is a bug that we must have introduced recently (since this used to work). I'm still trying to figure what happened and how to prevent it in the future.

**Comment by pastewka** [comment]
I've found the issue, it will be fixed with the next update. Will notify you here when that is deployed.

**Comment by pastewka** [comment]
@sitangshugk95 - I've just updated the platform, can you check if this works again?

**Comment by sitangshugk95** [comment]
This works now @pastewka , thanks a lot :)

**Comment by pastewka** [comment]
Thanks for reporting this @sitangshugk95
